lms/views/basic_lti_launch.py

```diff
@@ -335,27 +310,64 @@ def _set_via_url(self, document_url):
         self.context.js_config["urls"].update(
             {"via_url": via_url(self.request, document_url)}
         )
-        self._set_submission_param("document_url", document_url)
+        self.set_canvas_submission_param("document_url", document_url)
+
+    # ---------------------------------------------------------------------- #
+    # Canvas specific functions
```

Comment:
I'd vote we delete these kind of comments -- it's clear from the `canvas` in the names of all the methods below that they're Canvas-specific, and a comment like this is just begging for something not Canvas-specific to be added beneath it and unwittingly turn the comment into a lie.

Reply:
I guess I was trying to prevent that from happening by putting a big marquee, but I'll kick it out.

Reply:
Gone